src/common/utils/position.ts

```diff
@@ -26,6 +26,13 @@ const scaleBandInvert = (scale) => {
   };
 };
 
+export const getSelectedSegment = (position: number, data, positionCoord = 'x') => {
+  // return the item in data that the mouse position is hovering on
+  return data.find((item, i) =>
+    item[positionCoord] <= position && (data[i + 1]?.[positionCoord] > position || data[i + 1] === undefined)
+  );
+};
+
```

Comment:
was thinkin about maybe moving this to `utils/selection.ts`? I'll prob keep it here unless anyone has a stronger opinion

Reply:
Fine w/ me - Let's add some docs to this function on what exactly it does.

Reply:
Removed this in favor of adding functionality to getClosestPosition()